perun-core/src/main/java/cz/metacentrum/perun/core/impl/ResourcesManagerImpl.java

```diff
@@ -1183,6 +1194,32 @@ public void removeBan(PerunSession sess, int memberId, int resourceId) throws Ba
 		}
 	}
 
+	@Override
+	public List<AssignedResource> getResourceAssignments(PerunSession sess, Group group) {
+		try {
+			return jdbc.query("select " + assignedResourceMappingSelectQuery + " from resources" +
+					" join groups_resources on resources.id=groups_resources.resource_id" +
+					" where groups_resources.group_id=?", ASSIGNED_RESOURCE_MAPPER, group.getId());
+		} catch (EmptyResultDataAccessException e) {
+			return new ArrayList<>();
```

Comment:
This exception cannot be actually thrown by the `jdbc.query` method. It can be thrown by `jdbc.queryForObject` for example. Same for other methods.

Reply:
Removed